rescript/plugin_setup.py

```diff
@@ -828,6 +830,34 @@
         "used for slicing the original alignment."),
 )
 
+T = TypeMatch([AlignedSequence, Sequence])
+plugin.methods.register_function(
+    function=subsample_fasta,
+    inputs={'sequences': FeatureData[T], },
+    parameters={
+        'subsample_size':
+            Float % Range(0, 1, inclusive_start=False, inclusive_end=True) |
+            Int % Range(1, None),
+        'random_seed': Int % Range(1, None)
+    },
+    outputs=[('sample_sequences', FeatureData[T]), ],
+    input_descriptions={'sequences': 'Sequences to subsample from.', },
+    parameter_descriptions={
+        'subsample_size': 'Size of the sample: if a number 0.0-1.0 is given '
```

Comment:
I recommend changing the description to:

"Size of the random sample. Either a fraction of the total count (if a float) or a positive number (if an integer)."

No need to list the ranges in the description, as this information will be reported automatically.